linkerd/app/test/src/resolver.rs

```diff
@@ -29,6 +32,16 @@ pub type Dst<E> = Resolver<NameAddr, DstReceiver<E>>;
 
 pub type Profiles = Resolver<Addr, Option<profiles::Receiver>>;
 
+#[cfg(feature = "client-policy")]
+pub type ClientPolicies = Resolver<Addr, watch::Receiver<client_policy::ClientPolicy>>;
```

Comment:
Is it worth sticking this stuff into a new file/submodule? We can minimize cfg blocks to the top level module

Reply:
yeah, i also thought about this, didn't do it because there were just a couple impl blocks, but then it rapidly grew in size...